sapp/ui/frontend/src/Issue.js

```diff
@@ -239,7 +241,9 @@ export function Issue(
       <Row gutter={gutter}>
         <Label>Code</Label>
         <Item>
-          <Text code>{props.issue.code}</Text>
+          <Tooltip title={props.issue.warning_message}>
+            <Text code>{props.issue.code}</Text>
```

Comment:
```suggestion
            <Text code>{props.issue.code} : {props.issue.warning_message}</Text>
```

Reply:
Thanks @0xedward 
when I followed this suggestion 

it looked like this 👇 
![Screenshot (137)](https://user-images.githubusercontent.com/57163971/137893850-e7fc54cd-9048-4928-a4f3-0b586d427807.png)

being on the same row looked somehow
so I added it on a different line 
Like this  👇 
![Screenshot (135)](https://user-images.githubusercontent.com/57163971/137893836-c30c85ed-927d-43ab-a754-b3a189ae6787.png)

which do you think is better?

Reply:
@esiebomaj Thanks for checking! Not sure what others prefer, but I think we should go with `6073: Potential Server Side Template Injection` for `Code` here (i.e. the option were we don't create a new row for warning message description)